src/components/dashboard/sidebar/EnterpriseOffersSummaryCard.jsx

```diff
@@ -9,14 +9,16 @@ import {
 } from './data/constants';
 import SidebarCard from './SidebarCard';
 
+function getOfferExpiringFirst(offers) {
+  return offers
+    .filter(offer => offer.isCurrent)
+    .sort((a, b) => new Date(a.endDatetime) - new Date(b.endDatetime))[0];
```

Comment:
Will this throw an exception if offers is empty?

Reply:
No. Getting an index from an array that has 0 elements does not throw an exception; it just returns `undefined`. However, where we read the result from this function could throw an exception if it returns `undefined`. Fixing in follow-up PR.

<img width="105" alt="image" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/2828721/7d9b90e6-9783-4ca2-9fce-9a38e883eb07">